.circleci/config.yml

```diff
@@ -1231,9 +1231,20 @@ jobs:
       - update-spm-installation-commit
       - run:
           name: Submit Purchase Tester
-          working_directory: "Tests/TestingApps/PurchaseTesterSwiftUI"
           command: bundle exec fastlane deploy_purchase_tester
 
+  deploy-purchase-tester-dry-run:
+    <<: *base-job
+    steps:
+      - checkout
+      - setup-git-credentials
+      - trust-github-key
+      - install-dependencies
+      - update-spm-installation-commit
+      - run:
+          name: Submit Purchase Tester
+          command: bundle exec fastlane deploy_purchase_tester dry_run:true
```

Comment:
Hmm I guess the `dry_run` could be a parameter of the job... But maybe this is better so it's more delimited, I'm ok keeping it as is.

Reply:
I think it should be a parameter, main reason is that if we update the `deploy-purchase-tester` job, we will probably forget about updating this other job. Having a single job with a parameter will make sure we only have to update one thing

Reply:
That's a great point. It's slightly tricky because of how we re-use jobs, but it's still very easy. I'll open up a separate PR for it since this one is approved and so that we can have a specific conversation about it